app/src/main/kotlin/edu/stanford/bdh/engagehf/bluetooth/data/models/BluetoothUiState.kt

```diff
@@ -11,3 +13,15 @@ sealed interface BluetoothUiState {
     // Generic error
     data class Error(val message: String) : BluetoothUiState
 }
+
+data class MeasurementDialogUiState(
+    val measurement: Measurement? = null,
+    val isVisible: Boolean = false,
+    val isProcessing: Boolean = false,
```

Comment:
💪 

Very nice. Relates to #39 and it's the correct way to handle loading states